### ISSUE TITLE
Photopea for tablets 

### ISSUE THREAD

**Issue by Abeelme** [photopea/photopea#5108]
I like to see photopea with new interface and ui for tablets like photoshop for iPad os 
The current interface it's not good for touch devices 
I hope to make my dream come true 😊
And how about a native app for android 
There is no good graphic design app for android unlike iPad os like photoshop for iPad affinity designer, photo ...
You can be the hero for android users
https://helpx.adobe.com/photoshop/using/workspace-ipad.html

**Comment by photopea** [comment]
Did you try Photopea on your iPad? What exactly would you like to be different? Or are you just asking for a "different app", with 10 features instead of 1000? :D

**Comment by photopea** [comment]
What exactly is hard and what would you like to be different? How different?

**Comment by Abeelme** [comment]
I want to use photopea without mouse and keyboard on my android device 
So I want a new UX for example photoshop on desktop is different on iPad 
Affinity photo also is different on ipad 
They use more touch shortcut 
The ui is also different is more suitable for ipad 
I want to bring this experience to android devices specially for tablets 
You can see this video on YouTube to get my idea
https://youtu.be/nudhIktz-3Q

**Comment by photopea** [comment]
I think Photopea on tablets is also easy to use, there are touch shortcuts, too.

**Comment by MaxPuig** [comment]
Just today I tried Photopea on iPad with an Apple Pencil but without keyboard or mouse. The one problem I encountered is that there is no way to paste an image, or at least I couldn't find it.

Another thing I didn't quite like, but completely understand that it exists, is the Ad space. The iPad I use has an 4:3 aspect ratio and it takes up a big space. Maybe a periodic non-intrusive pop-up would be a good alternative?

What I really like is that when adding the website to the homescreen, it works as an independent app. 
I've been using this tool on a PC lately and it works perfectly. Great job!
